R/data-objects.R

```diff
@@ -205,15 +221,16 @@ local_to_irods_ <- function(
   irods_rest_call("stream", "PUT", args, verbose, object)
 }
 
-#' Retrieve file or object from iRODS
+#' Retrieve File or Object from iRODS
 #'
 #' Transfer a file from iRODS to the local storage with `iget()` or
-#' read an R object from an RDS file in iRODS with `ireadRDS()` (see `readRDS()`).
+#' read an R object from an RDS file in iRODS with `ireadRDS()`
```

Comment:
`[ireadRDS()]` instead of `` `ireadRDS()` ``

Reply:
done